microsoft/devhome
# Project Ironside: WinLogs Page - Sort logs based on timestamp and enable text filtering

## Summary of the pull request
1. Rename WinLogsHelper to WinLogsService and make it a Singleton
2. Use CollectionViewSource to sort and filter logs

## Validation steps performed
1. Launch PI
4. Attach to ProblemApp2
5. Go to WinLogs page
6. Check logs are in ascending order
7. Enter text in the text box to filter logs

## PR checklist
- [ ] Closes #3073
